jsign-core/src/main/java/net/jsign/asn1/authenticode/SpcLink.java

```diff
@@ -54,7 +54,7 @@ public ASN1Primitive toASN1Primitive() {
         } else if (moniker != null) {
             return new DERTaggedObject(false, 1, moniker);
         } else {
-            return new DERTaggedObject(false, 2, file);
+            return new DERTaggedObject(false, 2, new SpcString(""));

```

Comment:
What about changing the `file` field instead?

Reply:
That would be better yes. Will update.

Reply:
Updated in one of the last commits.